### PR TITLE
[Fix #201] Removing old elasticsearc compatibility

### LINKED ISSUE #201: Remove old elasticsearch versions compatibility
Is there a reason for this type of compatibility code? If every version only supports the corresponding major one I don't see a reason not to remove it, specially the ES2 one.
https://github.com/sabricot/django-elasticsearch-dsl/blob/addce06166dd31a052316e07b44736b451779f75/django_elasticsearch_dsl/fields.py#L213-L252

I'll be happy to make a pull request to at least remove it from 7.x branch

### PR DESCRIPTION
**Pull request by safwanrahman**
This fixes  #201

### CODE REVIEW

**Comment by alexgmin** [comment]
Can you also update the docs?

File: django_elasticsearch_dsl/fields.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import collections
 from types import MethodType
-import warnings
 
+from elasticsearch_dsl.field import Keyword, Text
```
Comment (alexgmin) [inline django_elasticsearch_dsl/fields.py]: These imports should be together with the ones in line 11.

Comment (safwanrahman) [reply django_elasticsearch_dsl/fields.py]: Thanks!

**Comment by safwanrahman** [comment]
@alexgmin This has been updated already. in documentation it is mentioed about which version to install for which ES version.

**Pull request merged**